src/cart/crt_hg_proc.c

```diff
@@ -207,39 +207,19 @@ int
 crt_proc_d_string_t(crt_proc_t proc, d_string_t *data)
 {
 	hg_return_t	hg_ret;
-	hg_proc_op_t	 proc_op;
-
-	proc_op = hg_proc_get_op(proc);
```

Comment:
those changes were specifically added by Ashley recently in order for log parsing to work properly. will need his review/feedback on it